ui/webapp.py

```diff
@@ -45,12 +45,19 @@ def reset_results(*args):
         state.raw_json = None
 
     # Title
-    st.write("# Haystack Demo")
+    st.write("# Haystack Demo - Explore the world")
+    st.write("""
+This demo takes its data from a selection of Wikipedia pages crawled in November 2021 on the topic of 'Countries and capital cities'. 
+
+Ask any question on this topic and see if Haystack can find the correct answer to your query! 
+
+*Note: do not use keywords, but type full-fledged questions.* The demo is not optimized to deal with keyword queries and might misunderstand you.
+""")
 
     # Sidebar
     st.sidebar.header("Options")
-    top_k_reader = st.sidebar.slider("Max. number of answers", min_value=1, max_value=10, value=3, step=1)
-    top_k_retriever = st.sidebar.slider("Max. number of documents from retriever", min_value=1, max_value=10, value=10, step=1)
+    top_k_reader = st.sidebar.slider("Max. number of answers", min_value=1, max_value=10, value=3, step=1, on_change=reset_results)
+    top_k_retriever = st.sidebar.slider("Max. number of documents from retriever", min_value=1, max_value=3, value=10, step=1, on_change=reset_results)
```

Comment:
`value=10` should be `value=3` to make it less or equal to the `max_value`

Reply:
oops good catch!